frankly-js/FranklyClient.html

```diff
@@ -73,7 +73,7 @@ <h4 class="name" id="FranklyClient"><span class="type-signature"></span>new Fran
 with the same or different pairs of appKey and appSecret.
 
 Each instance of FranklyClient maintains its own connection pool to Frankly
-servers, if the client is not required anymore then the application should
+servers, if the client is not required anymore then the application should 
```

Comment:
This file shouldn't be here, it has been updated on master already and this is an old version which is included in this pull request (I guess there are more like that for frankly-python and frankly-ruby).